.github/workflows/build-wheel-linux-x86_64.yaml

```
@@ -416,7 +417,12 @@ jobs:
 
     - name: Install Catalyst
       run: |
-        python${{ matrix.python_version }} -m pip install $GITHUB_WORKSPACE/dist/*.whl --extra-index-url https://test.pypi.org/simple
+        python${{ matrix.python_version }} -m pip install pennylane-lightning==0.36.0
+        python${{ matrix.python_version }} -m pip install 'pennylane-lightning[kokkos]==0.36.0'
+        python${{ matrix.python_version }} -m pip install --upgrade git+https://github.com/PennyLaneAI/pennylane@v0.36.0-rc0
+        python${{ matrix.python_version }} -m pip install jax==0.4.23 jaxlib==0.4.23 tomlkit 'scipy<1.13' 'numpy<2' 'diastatic-malt>=2.15.1'
+        python${{ matrix.python_version }} -m pip install $GITHUB_WORKSPACE/dist/*.whl --no-deps
+
```

Let's add some todo's next time so it's always clear what is a temporary fix :)
```suggestion
        python${{ matrix.python_version }} -m pip install $GITHUB_WORKSPACE/dist/*.whl --extra-index-url https://test.pypi.org/simple
```